code-challenges/llMerge/ll-merge.test.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const linkedList = require('./ll-merge.js');
+
+const firstLinkedList = new linkedList.LinkedList();
+const secondLinkedList = new linkedList.LinkedList();
+const mergeList = new linkedList.LinkedList();
+
+firstLinkedList.insert('2');
+firstLinkedList.insert('3');
+firstLinkedList.insert('1');
+
+secondLinkedList.insert('4');
+secondLinkedList.insert('9');
+secondLinkedList.insert('5');
+
+
+describe('Testing Merged Linked Lists', () => {
+  it('Should zipper two linked lists together by taking in 3 linked list variables', () => {
+    // console.log(linkedList.mergeLists());
+    expect(linkedList.mergeLists(firstLinkedList, secondLinkedList, mergeList)).toStrictEqual(` { 4 } { 2 } { 9 } { 3 } { 5 } { 1 }`);
+  });
```

Comment:
More tests that you could write for this would be:
- What happens if one of the linked lists is empty
- What happens if one of the linked lists is shorter than the other